fe/fe-core/src/main/java/com/starrocks/privilege/PrivilegeManager.java

```diff
@@ -306,6 +331,93 @@ public short analyzeType(String typeName) throws PrivilegeException {
         return typeStringToId.get(typeName);
     }
 
+    public void createRole(CreateRoleStmt stmt) throws DdlException {
+        roleWriteLock();
+        try {
+            String roleName = stmt.getQualifiedRole();
+            Long roleId = getRoleIdByNameNoLock(roleName);
+            if (roleId != null) {
+                throw new DdlException(String.format("Role %s already exists! id = %d", roleName, roleId));
+            }
+            RolePrivilegeCollection collection = new RolePrivilegeCollection(roleName);
```

Comment:
so we don't support create role with some initial priviledges? we can only grant roles or priviledges to a role after creation?

Reply:
No, we don't. Neither will we support creating a user with initial privileges. PM had a discussion earlier and decided it's better this way.

Reply:
ok